src/main/java/org/openrewrite/java/testing/junit5/UpdateTestAnnotation.java

```diff
@@ -121,7 +121,7 @@ public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration method, Ex
                             .visitNonNull(m, ctx, getCursor().getParentOrThrow());
                 }
                 if (cta.expectedException != null) {
-                    m = JavaTemplate.builder("Object o = () -> #{};")
+                    m = JavaTemplate.builder("org.junit.jupiter.api.function.Executable o = () -> #{};")
                             .contextSensitive()
```

Comment:
I haven't tested or debugged this, but I would have expected the template to require a parser with JUnit on the classpath, so that the type attribution is correct.

Reply:
Do you think it's really needed here since we are throwing away the variable and just keeping the lambda expression and "re-assigning" the type couple of lines after?

Reply:
I think the error you saw was already solved by a commit that I had made: I added a missing semicolon at the end. But the fact that the variable type isn't really compatible with a lambda in the initializer makes me wonder what we get as type attribution. I will take a closer look tomorrow.

Reply:
You are completely right, I did not realize about the extra semicolon between 2.0.2 and the last commit on main... and that's the reason why it's working fine, not the functional interface... I'm closing the PR